protocols/cosi/packets.go

```diff
@@ -11,6 +11,12 @@ import (
 
 // The main messages used by CoSi
 
+// see https://github.com/dedis/cothority/issues/260
+// 0 - no check at all
+// 1 - check only at root
+// 2 - check at each level of the tree
+var VerifyResponse = 1
```

Comment:
this is not a const. 
Linus made this change here:
https://github.com/dedis/cothority/pull/264/commits/a04ab0a85f14fd67c1884ef03c63cf3ff4e9f67d
so it's not passed as a message. Anyway it's for the simulation so..